### DISCUSSION TITLE
How do we handle VSCode versions?

### DISCUSSION THREAD

**Discussion by fwendland** [Fraunhofer-AISEC/codyze#281] [General]
We just added a plugin for VSCode to use Codyze. Seeing the automated PR for dependency updates, we should decide, how to handle plugin versions.

- Do we maintain multiple versions of the VSCode plugin?
- When do we update to new versions of VSCode (e.g. `engines.vscode`)?
- Is there a method to link versions between dependencies, e.g. `engines.vscode` and `@types/vscode`?`

**Comment by oxisto** [comment]
Unfortunately, renovate does not seem to automatically update `engines.vscode`, which is slightly annoying :(

We should only support the most recent VS Code Version.

**Reply by fwendland** [reply]
Renovate seems to support `engines.vscode` and `@types/vscode`. See: https://github.com/renovatebot/renovate/pull/7468

No idea, why it didn't work.
